experiments/pages/sampleStyleGuidePage/README.md

```diff
@@ -0,0 +1,5 @@
+# Sample Style Guide Page
+
+_Better README coming soon_
```

Comment:
@hyork Can we go ahead and add the README here? I'm not entirely sure what the purpose/usage of this is without digging in.

Reply:
makes sense! will do